test/scripts/products/h22-validate-that-rate-limit-service-is-working-as-expected/h22-validate-that-rate-limit-service-is-working-as-expected.go

```diff
@@ -297,7 +331,6 @@ func createAPI(ctx context.Context, client k8sclient.Client, namespacePrefix, ba
 	if err != nil {
 		return nil, err
 	}
-
 	accessToken := string(s.Data["ADMIN_ACCESS_TOKEN"])
 	fmt.Printf("  🔑  Found access token: %s\n", accessToken)
 
```

Comment:
This comment applies to the line below it but I can't comment directly on line 337:

`resources.GetRhmiCr()` has been [deprecated](https://github.com/carlkyrillos/integreatly-operator/blob/master/pkg/resources/rhmi.go#L14) and `rhmi.GetRhmiCr()` should be used [instead](https://github.com/carlkyrillos/integreatly-operator/blob/master/pkg/resources/rhmi/rhmi.go#L13). The functionality is the exact same so switching it shouldn't require any further changes.

Reply:
replaced